### ISSUE TITLE
Finder Fee should be available in select/init requests too

### ISSUE THREAD

**Issue by mystore-in** [Open-network-for-digital-commerce/Pre-production#51]
Currently the buyer_app_finder_fee_type  is present in search requests as per the document but not in select and init requests.

https://docs.google.com/document/d/1osOKwHnpGrWoXAMONu8WXJ2AFOetyiRG7yYEO_gmRBU/edit

This requires search transactions to be linked with select/init transactions which is not practical as search results are mostly cached.

buyer_app_finder_fee_type must be present in select/init requests so that they are not linked to the search requests.

**Comment by BLR-0118** [comment]
finder fee cannot be in the select/init requests as it's used to compute the declared price shown to the buyer. You can map the finder fee with the corresponding buyer app and it's not necessary to link the search transaction with select/init.
@mystore-in

**Comment by mystore-in** [comment]
But finder fee can be and will be different for different categories of products. It can not be same for grocery and jewellery. I think it will be harmless to allow finder fee in select/init requests too so seller app do not have to link the  fee with the buyer app or make assumptions.

**Comment by BLR-0118** [comment]
Agree. There will be a subdomain in Context in future using which you can map finder fee for specific buyer app for domain & subdomain
